src/RestControllers/PatientRestController.php

```diff
@@ -75,7 +75,7 @@ function getOne()
     /**
      * Returns patient resources which match an optional search criteria.
      */
-    function getAll($search = array())
+    public function getAll($search = array())
```

Comment:
Thank you for adding these back in! I will certainly be more attentive the next time around!